config/creatrs/imp.cfg

```diff
@@ -127,10 +124,30 @@ TortureKind = NULL
 ;   NO_TRANSFER - Creature cannot be transferred with a transfer creature special.
 ;   NO_STEAL_HERO - Creature cannot be stolen from a steal hero special.
 ;   PREFER_STEAL - Creature can be generated from steal hero special if there is nothing to steal from.
-;   IMMUNE_TO_DISEASE - Creature cannot get diseased.
 ;   ILLUMINATED - A bright light will shine from the Creature.
 ;   ALLURING_SCVNGR - When scavenging will give the keeper a portal boost compared to rival keepers.
 Properties = BLEEDS SPECIAL_DIGGER EVIL NO_ENMHEART_ATTCK NO_CORPSE_ROTTING
+; Creature immunities:
```

Comment:
This is far too verbose. 

How about:

; Creature immunities. Field takes SpellFlags, see magic.cfg for the spellflags.
;  Special cases:
;    128: LIGHT - Prevents 'LIGHT', but does not stop 'ILLUMINATED' from working.
;   1024: POISON_CLOUD - Prevents damages/status from effects with 'AreaAffectType' set to 1/2/3.
;   4096: MAD_KILLING - Prevents 'MAD_KILLING', but does not stop 'MAD_PSYCHO' anger job.
;   8192: FEAR - Prevents 'FEAR', but does not stop fleeing/real fear.
; 131072: WIND - Prevents the creature from being blown away from shots with 'UpdateLogic' set to 2.
; Set to 0, leave empty, or remove the field for the creature to have no immunities.